config/chmo.yml

```diff
@@ -6,7 +6,20 @@ base_url: /obo/chmo
 products:
 - chmo.owl: https://raw.githubusercontent.com/rsc-ontologies/rsc-cmo/master/chmo.owl
 - chmo.obo: https://raw.githubusercontent.com/rsc-ontologies/rsc-cmo/master/chmo.obo
-
-term_browser: ontobee
+- chmo.json: https://raw.githubusercontent.com/rsc-ontologies/rsc-cmo/master/chmo.json
+- chmo-base.owl: https://raw.githubusercontent.com/rsc-ontologies/rsc-cmo/master/chmo-base.owl
```

Comment:
Projects may only use their IDSPACE under 'products'. So 'chmo.owl' is fine, but 'chmo-base.owl' and 'chmo-full.owl' are not (and likewise for '.obo' and '.json' formats).